src/pages/Dashboard.tsx

```diff
@@ -20,7 +23,10 @@ function Dashboard() {
     });
   }, []);
 
-  return <NoOrganization />;
+  if( organizationCount === 0 )
+    return <NoOrganization/>;
+
+  return <></>;
```

Comment:
Might be a little cleaner to render like so

```suggestion
  return organizationCount === 0 ? <NoOrganization/> : <></>;
```

Reply:
Dang those ternary operators are clean, why do I always neglect to use them.